src/components/SideNav/SideNav.jsx

```diff
@@ -12,36 +12,27 @@ const SideNav = () => {
     <aside className="side-nav">
       <ul className="nav-items">
         <li>
-          <NavLink to="/" exact activeClassName="active" className="nav-pill">
+          <NavLink to="/" activeclassname="active" className="nav-pill">
```

Comment:
Navlink by default adds an active class to your class list. You can directly access `nav-pill.active`

Reply:
Thanks for the info. I will change this